### ISSUE TITLE
Request: Dual Boot

### ISSUE THREAD

**Issue by erenoglu** [classy-giraffe/easy-arch#58]
**Describe the bug**
Script forces to erase all partition layout on disk, even if there's sufficient unused space on partition table, in a scenario where existing Windows would like to be kept intact for dual boot

**To Reproduce**
1. On your Windows PC, shrink the Windows Data partition and make available a large free space in partition table
2. Download Arch Iso and boot with it.
3. Use the easy-arch script, at least step, it will ask you to confirm deleting all your partitions

**Expected behavior**
Present dialog that new partitions will be created for boot and encrypted root, use default size for boot ie 2gb and allocate rest of free space to root.

**Comment by erenoglu** [comment]
yes it can be endless. My recommendation would be to start with simplest case which shall cover 95% of the setups, ie single disk,  Windows installed by default in factory, user pre-shrinked Windows and created empty space. So, you have EFI partition already to host ReFind or Grub, then create a 1GB boot partition (if required for easy encryption), and then create another partition for the rest of the system.

**Comment by classy-giraffe** [comment]
Could try to come up with something indeed. I also usually don't like to have linux and windows share the same EFI partition cause it happened to me multiple times that Windows deleted GRUB files and I ended up with a non functioning instance. Of course, if you just have one disk this is all you can do, unfortunately...
